hockeysdk/src/main/java/net/hockeyapp/android/tasks/AttachmentDownloader.java

```diff
@@ -126,7 +126,7 @@ public boolean hasRetry() {
         }
 
         public boolean consumeRetry() {
-            return --remainingRetries < 0 ? false : true;
+            return --remainingRetries >= 0;
```

Comment:
the previous code had pretty bad readability. The new one is better, but not much. Maybe we can be a little more expressive than `--remainingRetries >=0`?